src/dcat-us/dataset-formatter.test.ts

```diff
@@ -24,14 +24,14 @@ it('dcatHelper: it does not allow customizations to overwrite critical fields',
   expect(template['@type']).not.toBe('SABOTAGE');
   expect(template.contactPoint['@type']).not.toBe('SABOTAGE');
   expect(template.identifier).not.toBe('SABOTAGE');
-  expect(template.license).not.toBe('SABOTAGE');
   expect(template.landingPage).not.toBe('SABOTAGE');
   expect(template.webService).not.toBe('SABOTAGE');
   expect(template.spatial).not.toBe('SABOTAGE');
   expect(template.distribution).not.toBe('SABOTAGE');
   expect(template.title).toBe('{{metadata.metadata.name||item.title}}')
   expect(template.contactPoint.fn).toBe('{{item.owner}}');
   expect(template.contactPoint.hasEmail).toBe('mailto:dcat.support@dc.gov');
+  expect(template.license).toBe('{{CUSTOM LICENSE}}')
```

Comment:
I'd argue that you could take out references to license from this test entirely since you added 2 dedicated ones down below. Also might reduce future confusion since this test is meant to verify scrubbing.

Reply:
Fair point. Removed.